decide_panel/src/components/Bar.js

```diff
@@ -0,0 +1,55 @@
+import Api from "../services/backend";
+import React from "react";
+import { Bar } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
+
+
+const BarChart = () => {
+  return <div>
+      <Bar
+        data={{
+            labels: ["Opcion 1", "Opcion 2"],
+            datasets: [{
+
+                label: "Votos",
+                data: [1,1],
+                backgroundColor: [
+                    'rgba(255, 99, 132, 0.2)',
+                    'rgba(54, 162, 235, 0.2)'
+                    
+                ],
+                borderColor: [
+                    'rgba(255, 99, 132, 1)',
+                    'rgba(54, 162, 235, 1)'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(54, 162, 235, 1)"
```